### ISSUE TITLE
Install fails on alpine

### ISSUE THREAD

**Issue by rumpelsepp** [dustymabe/vagrant-sshfs#93]
I am using a alpine linux guest and this plugin fails with:

```
==> default: Installing SSHFS client...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

yum -y install fuse-sshfs

Stdout from the command:



Stderr from the command:

bash: line 4: yum: command not found
```

Please consider a switch not to install this plugin automatically, I do this in my provision script anyway. On alpine one would install sshfs with `apk add sshfs`.

**Comment by dustymabe** [comment]
what's the alpine linux vagrant box you are using? preferably one from atlas/vagrantcloud

**Comment by rumpelsepp** [comment]
I tried these:

https://app.vagrantup.com/generic/boxes/alpine36
https://app.vagrantup.com/generic/boxes/alpine37

**Comment by dustymabe** [comment]
hey @rumpelsepp - the way vagrant-sshfs detects a guest type is by re-using the guest detections that are defined in the files under:  https://github.com/hashicorp/vagrant/tree/master/plugins/guests . vagrant-sshfs can't support alpine until it is added there.

**Comment by timschumi** [comment]
I created a pull request to add alpine support (#105).

While upstream vagrant doesn't support the "alpine" guest type yet, it is provided by the vagrant-alpine plugin until it's ready to be upstreamed.

@rumpelsepp The specific issue with sshfs trying to install through `yum` is due to the guest type being hardcoded to `alt` as a workaround. If you have the vagrant-alpine plugin installed, add `config.vm.guest = :alpine` to your Vagrantfile to set it to the correct guest type. In conjunction with the PR I linked earlier, that should allow SSHFS to be installed automatically in the future.
